assets/css/style.scss

```diff
@@ -30,6 +30,22 @@ main.page-content {
   }
 }
 
+.footer-col-wrapper {
+  color: #111;
+}
+
+a, a:link, a:visited {
```

Comment:
Any particular reason you're making visited links the same color? I actually like them being different.

Reply:
Not really. Do you have a good color in mind?

Reply:
`#8E6680`?